Document Docker Desktop alternatives for Linux (#87)
eclipse-velocitas/velocitas-docs: content/en/docs/tutorials/quickstart/container_runtime.md

In the past the recommended runtime would for sure be [Docker Desktop](https://www.docker.com/products/docker-desktop). But since Docker Inc. changed their license model it is fair enough for an open source project to look for free alternatives.

## Linux

> The obvious (and our recommended) alternative to Docker Desktop on Linux is [Docker Engine](https://docs.docker.com/engine/), a pure CLI-based solution available for most popular Linux distributions licensed under the Apache License, version 2.0. Installation instructions can be found [here](https://docs.docker.com/engine/install/).

Reviewer: isn't the docker engine also part of docker desktop? So basically Docker Engine is not really an alternative but you can use it without Docker Desktop or? If so please adapt this section accordingly.

That's all you have to do. After these few steps you can go on with the devcontainer setup.

#### Drawbacks

The only drawback noticed so far is, that K9S is not working properly on M1 Macs. Since the container runtime and deployment are working also without K9S, this is just a minor issue. Nevertheless, the team is working on a solution.

> ## Microsoft Windows

Reviewer: Should we also mention the possibility to setup a virtual machine and use the linux way?

There is currently no recommended alternative for Windows except using GitHub codespaces, a cloud-based development environment.

## Other alternatives

> Besides our recommendations above there are further alternatives, that were not evaluated by us so far or we decided, not to give our recommendation so far.

Reviewer: What about something like this?
```suggestion
Besides our recommendations above, there are further alternatives, which are not yet evaluated or have some other drawbacks, blocking a recommendation. 
```
